docs/source/python.rst

```diff
@@ -0,0 +1,80 @@
+ome_zarr python library
+=======================
+
+writing OME-NGFF images
+-----------------------
+
+The principle entry-point for writing OME-NGFF images is :py:func:`ome_zarr.writer.write_image`.
+This takes an n-dimensional `numpy` array and writes it to the specified `zarr group` according
+to the OME-NGFF specification.
```

Comment:
Here it is called OME-NGFF specification. On the top of the main readthedocs page https://ome-zarr-py--121.org.readthedocs.build/en/121/intro.html#license it is called ome-zarr-spec.  Possibly this naming could be unified within one single doc here ?

Reply:
I've updated `ome-zarr-spec` to OME NGFF spec.